bots/pr/src/main/java/org/openjdk/skara/bots/pr/IntegrateCommand.java

```diff
@@ -196,20 +229,9 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
             // Rebase and push it!
             if (!localHash.equals(PullRequestUtils.targetHash(pr, localRepo))) {
                 var amendedHash = checkablePr.amendManualReviewers(localHash, censusInstance.namespace(), original);
+                addPrePushComment(pr, amendedHash, rebaseMessage.toString());
                 localRepo.push(amendedHash, pr.repository().url(), pr.targetRef());
```

Comment:
If the push fails and the bot dies, will it still be retried after this fix?

Reply:
Yes, that's the idea at least, but I should write a test for it. If the push fails, the next time around, the prepush comment is found, but the hash is not found in the target, so the IntegrationCommand will continue as if nothing has happened.